platform/mk2/hal/gps_skytraq/gps_device_skytraq.c

```diff
@@ -29,9 +29,10 @@
 #define MESSAGE_TYPE_BINARY		2
 #define GNSS_NAVIGATION_MODE		GNSS_NAVIGATION_MODE_AUTOMOBILE
 
-#define BAUD_RATE_COUNT 2
+#define BAUD_RATE_COUNT 3
 #define BAUD_RATES 	{			\
 		{921600, 8},			\
+		{115200, 5},            	\
```

Comment:
Here's my RCT after 60 seconds unplugged:

GPS: provisioning attempt
GPS: probing baud rate: 921600
GPS: Using baud rate: 921600
GPS: module detected at: 921600
GPS: detected current update rate: 10
GPS: Configuring NMEA messages
GPS: Message was successfully received.
GPS: Successfully configured NMEA
GPS: configure message type: win
GPS: configure navigation data message interval: win
GPS: Configuring Gnss Navigation Mode: win
GPS: Disabling NMEA messages
GPS: Message was successfully received.
GPS: Successfully disabled NMEA messages
GPS: provisioned


I think the module in RCT might have a large capacitor or battery that might be having it hold on to the configuration, as the setup commands we're sending are to have it store to NVRAM.

data sheet indicates 9600 baud default.  

it's harmleless to include the 115200 for probing, but freshly built RCT's with modules that have never been programmed in the past will detect first detect 9600, then likely will lock in at 921600 unless they've been cold for a long time, I think.

If we include 115200, I would have the order of detection be 9600, 921600, and 115200.